submissions/karelskiy/HTML and CSS practice/script.js

```diff
@@ -0,0 +1,35 @@
+sec.addEventListener('click', function(e){
```

Comment:
This task was supposed to be done without JS.
Don't use JS in this task.

Reply:
Thank for comments, at first sorry about js, i didn't saw this. Also i want ask you, about this button, it should be hidden when i forcibly scrolling mouse? Or just click on button without scrolling?

Reply:
As I understand you right, this button must be present always, also, it must be accessible from the keyboard.